docs/content/Configuration/Config.mdx

```diff
@@ -30,7 +30,9 @@ interface CubejsConfiguration {
   basePath: string;
   webSocketsBasePath: string;
   logger: (msg: string, params: object) => any;
-  driverFactory: (context: DriverContext) => DriverConfig | BaseDriver | Promise<BaseDriver>;
+  driverFactory: (
+    context: DriverContext
+  ) => DriverConfig | BaseDriver | Promise<BaseDriver>;
```

Comment:
Promise<DriverConfig> is also supported.

Reply:
```suggestion
  ) => DriverConfig | BaseDriver | Promise<BaseDriver> | Promise <DriverConfig>;
```